src/Shared/Recruit.Shared.Web/Configuration/EnvironmentNames.cs

```diff
@@ -12,7 +12,7 @@ public static class EnvironmentNames
         public const string PREPROD = "PREPROD";
         public const string PROD = "PROD";
 
-        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, TEST, DEMO };
+        public static string[] GetTestEnvironmentNames() => new []{ Development, AT, DEMO };
```

Comment:
TEST is a test environment

Reply:
its just for testing. It will be put back later.

Reply:
why can't this be tested on PREPROD then. It's what we did for testing blocked employers. We shouldn't be making temporary code changes that go into master

Reply:
i will not be merging into master for now ... once testing is passed... i ll revert the changes that were environment specific.